### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by alanlowzies**
This PR contains the following additions/changes to the user guide:
- Add a warning to discourage users from directly modifying the local data files for the app.
- Change the warning (regarding the pipe character `|`) format to make it look more obvious.
- Add a list of recommended terminals to use.
- Add an FAQ on file loading issues.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -922,6 +939,13 @@ with the WerkIt! JAR file inside it, and run the application from the directory.
 When you launch WerkIt! in subsequent sessions, please ensure that you run it from the same directory
 that you did when starting WerkIt! for the first time.
 
+**Warning:** It is highly recommended that you do not directly modify these files on your own as it may lead to
+instabilities in the application. If you do decide to modify the files, **please make a backup of the `werkItResources`
+directory** before modifying the files. 
+
+In the event that your application becomes unstable due to accidental file modifications, please replace the
+`werkItResources` directory with your backup copy.
```
Comment (Musfirahe0556596) [inline docs/UserGuide.md:947]: or "can simply delete the files/folder in the worst scenario"?

**Review by Musfirahe0556596 — APPROVED** [review]
Nice job done!

File: docs/UserGuide.md (hunk 1 above)
Comment (alanlowzies) [reply docs/UserGuide.md]: Thank you for your feedback! It has been implemented in Commit 5e7ff2e

**Comment by alanlowzies** [comment]
> Nice job done!

Thank you :)

**Review by Musfirahe0556596 — APPROVED** [review]

**Pull request merged**